src/webgpu/api/operation/rendering/blending.spec.ts

```diff
@@ -621,6 +621,65 @@ g.test('simple_blend_constant,blend_constant_non_inherited')
     t.expectOK(result);
   });
 
+g.test('color_write_mask,blending_disabled')
+  .desc(`Test that the color write mask works when blending is disabled.`)
```

Comment:
nit: Disabling blending (`blend: undefined`) is subtly different from using the default blend state (which is what you have here, or equivalently `blend: { color: {}, alpha: {}`). Perhaps should test both?

(It makes sense to test this interaction, but just curious, how did you decide to test it?)

Reply:
Ah, this was my misunderstanding because I migrated the end2end test[1] to CTS. In the end2end test, it's using `srcFactor: one, dstFactor: zero, operation: add`. So, I understood that the blend component meant that blending is disabled :(. Sorry for my misunderstanding. Now I update the test to check both cases, one is undefined, the other one uses the blending component.    

[1] https://source.chromium.org/chromium/chromium/src/+/main:third_party/dawn/src/dawn/tests/end2end/ColorStateTests.cpp;l=749?q=ColorStateTest&ss=chromium

Reply:
No worries, it's confusing because earlier on in the design of the API we didn't make a distinction between "blend disabled" and "blend with default options", but we ended up deciding later to add it:
https://github.com/gpuweb/gpuweb/pull/1134
https://github.com/gpuweb/gpuweb/wiki/GPU-Web-2020-11-16#make-blending-state-explicit-1134

Reply:
Thanks for the information!